src/my-requests.js

```diff
@@ -349,6 +349,14 @@ class RequestList extends preact.Component {
                     } – ${request.reference})`
                 );
             });
+            const re = /\(.+\)$/;
+            let heading =
+                items.length <= 2
+                    ? `for ${items
+                          .map((item) => item.split(re)[0].trim())
+                          .slice(0, 2)
+                          .join(', ')}`
+                    : `for ${items.length} companies`;
```

Comment:
Instead of just showing the number here, it would be great to still list first one or two company names and then say "and <n> more companies" or something like that.

Reply:
okay done